wordpress-mobile/WordPress-iOS
# Untapabble help button on Sing Up

Sister PR to: https://github.com/wordpress-mobile/WordPress-iOS/pull/7913

To test:

- Launch the app.
- Enter the Sign Up flow and confirm you can tap on the help button.
- Confirm the Help button's layout is correct.

Needs review: @nheagy @aerych Can either of you pick this one? Thx
